src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -245,7 +246,7 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 		this._codeEditorService.removeDecorationType(key);
 	}
 
-	$getDiffInformation(id: string): Promise<ILineChange[]> {
+	$getDiffInformation(id: string): Promise<IChange[]> {
```

Comment:
@lszomoru I changed the return type here because this is what `getChanges` has always returned.